cgm/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+autoreconf -fi
+./configure --prefix=$PREFIX --enable-optimize --disable-debug --enable-shared
```

Comment:
Nope.  CGM is a common interface to these other geometry engines, so it doesn't really make sense to build without it.